gnd/build.gradle

```diff
@@ -56,7 +56,7 @@ android {
         // fix before increasing this value.
         targetSdkVersion 26
         versionCode gitVersioner.versionCode
-        versionName gitVersioner.versionName
+        versionName gitVersioner.versionName + " " + gitVersioner.currentSha1
```

Comment:
Is there a way to get the shortened commit ID instead? From the command line the latest commit can be shown with `git show -s --format=%h`.

Reply:
The git versioner plugin doesn't provide that out of the box. I'll try doing it directly like you suggested. Maybe we can remove this dependency too.

Reply:
The short version is just the first 7 characters of the commit hash. Is it possible to truncate to only include those?

Reply:
Done

Reply:
sample output : `7571-version+3 6714acd75`